Cargo.lock

```diff
@@ -391,6 +391,12 @@ dependencies = [
  "autocfg",
 ]
 
+[[package]]
+name = "once_cell"
```

Comment:
😅 omg I misremembered.

Artichoke _did_ have `once_cell` back when it had a lazily-allocated `Regex` backend but that got nuked a while ago. Either way, thanks for making this change. I prefer `once_cell` to `lazy_static`.

Reply:
once_cell does look a bit nicer I must say. It's a little more explicit which is better IMO